src/resources/handler.js

```diff
@@ -0,0 +1,37 @@
+Object.assign(pc, function () {
+    'use strict';
+
+    /**
+     * @interface pc.ResourceHandler
```

Comment:
What's this file for? It's not actually used by the sourcebase in this PR.

Reply:
The original jsdoc comments references pc.ResourceHandler as a type (in `resources/loader.js` and `resources/model.js`). But since there is no interface declaration for pc.ResourceHandler, the TypeScript declarations break.

I just now also added it to `dependencies.txt` so that it is included. This way devs can use it as a documented base class for their custom handlers (which we need).

Reply:
@daredevildave You wrote this code - what would you like to happen here?

Reply:
I think it needs a patch function too? But otherwise it seems fine.

Reply:
@daredevildave Good point. Patch function added.